lib/node_modules/@stdlib/ndarray/base/pop/lib/main.js

```diff
@@ -0,0 +1,87 @@
+/**
+* @license Apache-2.0
+*
+* Copyright (c) 2025 The Stdlib Authors.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+'use strict';
+
+// MODULES //
+
+var getShape = require( '@stdlib/ndarray/base/shape' );
+var sliceFrom = require( '@stdlib/ndarray/base/slice-from' );
+var sliceTo = require( '@stdlib/ndarray/base/slice-to' );
+
+
+// MAIN //
+
+/**
+* Returns an array containing a truncated view of an input ndarray and the last element along a specified dimension.
+*
+* @param {ndarray} x - input array
+* @param {integer} dim - dimension along which to perform the operation
+* @param {boolean} strict - boolean indicating whether to enforce strict bounds checking
+* @param {boolean} writable - boolean indicating whether a returned array should be writable
```

Comment:
```suggestion
* @param {boolean} writable - boolean indicating whether returned arrays should be writable
```

More than one ndarray is returned.